internal/certificateprovider/certificateproviderdata/provided.go

```diff
@@ -38,6 +38,16 @@ type Provided struct {
 	ContactLanguagePreference localize.Lang
 	// Email is the email address returned from OneLogin when the certificate provider logged in
 	Email string
+	// IdentityDetailsMismatched is set when details are updated to match confirmed identity
```

Comment:
`is set when confirmed identity does not match the details previously entered`?

Reply:
My bad, I wrote the comment when I'd misunderstood what the trigger for this was.